container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2428,6 +2437,7 @@ configuration:
     properties.capi.cc_uploader.mutual_tls.ca_cert: '"((INTERNAL_CA_CERT))"'
     properties.capi.cc_uploader.mutual_tls.server_cert: '"((CC_UPLOADER_CRT))"'
     properties.capi.cc_uploader.mutual_tls.server_key: '"((CC_UPLOADER_KEY))"'
+    properties.capi.nsync.lifecycle_bundles: '{"buildpack/((SUSE_STACK))":"buildpack_app_lifecycle/buildpack_app_lifecycle.tgz","buildpack/cflinuxfs2":"buildpack_app_lifecycle/buildpack_app_lifecycle.tgz","docker":"docker_app_lifecycle/docker_app_lifecycle.tgz"}'
```

Comment:
Huh, I'm surprised this works. The old opinions were a list of strings (which contains colons in them), not a hash.

That gets used for [bulker](https://github.com/cloudfoundry/capi-release/blob/50e96e2eeedaedb865099a87fee30508c0a86265/jobs/nsync/templates/nsync_bulker_config.json.erb#L35) and [listener](https://github.com/cloudfoundry/capi-release/blob/50e96e2eeedaedb865099a87fee30508c0a86265/jobs/nsync/templates/nsync_listener_config.json.erb#L29), which goes to [bulker](https://github.com/cloudfoundry/nsync/blob/6709e23af984d298b3217317b5b6078fb6017857/config/config.go#L62) and [listener](https://github.com/cloudfoundry/nsync/blob/6709e23af984d298b3217317b5b6078fb6017857/config/config.go#L79), which are both arrays of strings.